Kudu.Core/Scripts/kudusync

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-"$0.NET.exe" "$@"
+mono "$0.NET.exe" "$@"
```

Comment:
Yes, I think so. Note that to make things more confusing, we have 2 versions of kudusync: kudusync.net and kudusync (in different ProjectKudu repos). In Azure, we use kudusync.net, but I don't quite recall when the other one is used.